docs/node/run-your-node/prerequisites/system-configuration.mdx

```diff
@@ -127,24 +69,83 @@ Add the following task to your playbook:
 </TabItem>
 </Tabs>
 
-:::tip
+:::
+
+## Increase File Descriptor Limit
+
+Make sure that the user under which you are running your Oasis Node has a
+high-enough file descriptor limit as the databases can have many files open and
+running out of file descriptors can lead to the node stopping unexpectedly.
+
+You can check the file descriptor limit by running the following as the same
+user that will run Oasis Node:
+
+```bash
+ulimit -n
+```
+
+If this number is lower than 102400 you should consider increasing it by
+updating your system configuration. You can configure _temporary_ limits by
+running:
+
+```bash
+ulimit -n 102400
+```
+
+Note that this limit only applies to any processes started from the same shell
+after the command was executed. If you want to make the change _permanent_, you
+have the following options:
+
+<Tabs>
+<TabItem value="System config file">
```

Comment:
1. "System config file" sounds very generic. The config below is actually read by the PAM module during login and resides in the `limits.conf` file or `limits.d` folder. So I would rename the tab to this.

  ```suggestion
  <TabItem value="limits.conf">
  ```

2. Also, I would change the order and have `systemd` (lowercase), `Docker` and the new one so it's consistent with the tabs below.